packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -219,87 +279,12 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
     control,
     getValues,
     formState: { errors, isDirty },
-    setError,
   } = methods;
 
   useEffect(() => {
     promptBeforeClosing(() => isDirty);
   }, [isDirty, promptBeforeClosing]);
 
-  let [maxVisitStartDatetime, minVisitStopDatetime] = useMemo(() => {
-    if (!visitToEdit?.encounters?.length) {
-      return [null, null];
-    }
-
-    const allEncountersDateTime = visitToEdit?.encounters?.map(({ encounterDatetime }) =>
-      Date.parse(encounterDatetime),
-    );
-    const maxVisitStartDatetime = Math.min(...allEncountersDateTime);
-    const minVisitStopDatetime = Math.max(...allEncountersDateTime);
-    return [maxVisitStartDatetime, minVisitStopDatetime];
-  }, [visitToEdit]);
```

Comment:
This logic has been moved up to lines 128-140 and changed the variable names to better reflect what they actually do - `earliestEncounterDate` instead of `maxVisitStartDatetime` and `latestEncounterDate` instead of `minVisitStartDatetime`.